conf/nihbiowulf.config

```diff
@@ -0,0 +1,42 @@
+//Profile config names for nf-core/configs
+params {
+  config_profile_description = 'National Institutes of Health, USA: Biowulf nf-core config'
+  config_profile_contact = 'Kevin Brick (@kevbrick)'
+  config_profile_url = 'https://hpc.nih.gov/apps/nextflow.html'
+  max_memory = 224.GB	
+  max_cpus = 32
+  max_time = 72.h
+
+  igenomes_base = '/fdb/igenomes/'
+}
+
+process {
+  scratch = '/lscratch/$SLURM_JOBID'
+  maxForks = 100
+}
+
+profiles {
+  local {
+    process.executor = 'local'
```

Comment:
This is the default so you don't normally need to bother putting it in a config unless you're actively overwriting some other config / profile..